plumbing/object/change.go

```diff
@@ -61,6 +70,12 @@ func (c *Change) Files() (from, to *File, err error) {
 	return
 }
 
+func isInvalidFileMode(t TreeEntry) bool {
```

Comment:
Right. I was confused when we talked this morning.
@ajnavarro Symlinks can be returned by Files()